pkg/cloudprovider/providers/vsphere/vclib/vdmDiskManager.go

```diff
@@ -0,0 +1,52 @@
+package vclib
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/golang/glog"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+type vdmDiskManager struct {
```

Comment:
vdm itself stands for VirtualDiskManager.
Can I suggest 2 things:
1. Put all disk managers in vclib/diskManagers
2. Define one interface for diskManager havig APIs like create, delete etc. and vdm and vmDiskManager will implement them or throw exception if operation is not supported. e.g delete is not supported in vmDiskManager
2. rename vdmDiskManager.go to vdm.go